userspace/libscap/linux/scap_machine_info.c

```diff
@@ -122,24 +122,6 @@ static uint64_t scap_linux_get_host_boot_time_ns(char* last_err)
 	return 0;
 }
 
-static void scap_get_bpf_stats_enabled(scap_machine_info* machine_info)
-{
-	machine_info->flags &= ~PPM_BPF_STATS_ENABLED;
-	FILE* f;
-	if((f = fopen("/proc/sys/kernel/bpf_stats_enabled", "r")))
-	{
-		uint32_t bpf_stats_enabled = 0;
-		if(fscanf(f, "%u", &bpf_stats_enabled) == 1)
-		{
-			if (bpf_stats_enabled != 0)
-			{
-				machine_info->flags |= PPM_BPF_STATS_ENABLED;
```

Comment:
Please see the commit: https://github.com/falcosecurity/libs/pull/1401/commits/911b87ee56147d64af5199dc2a0bfa21c6aef2c7 (though it probably will get rebased to a different id), it's an engine feature now, rather than a machine feature (kmod doesn't care about BPF stats, for example)